pkg/node/node.go

```diff
@@ -241,15 +244,13 @@ func (n *Node) publishSignerPublicKey(
 		publicKey,
 	)
 
-	monitoringAbort := make(chan interface{})
-	go n.monitorKeepPublicKeySubmission(monitoringAbort, keepAddress)
-
 	err := n.ethereumChain.SubmitKeepPublicKey(keepAddress, publicKey)
 	if err != nil {
-		close(monitoringAbort)
 		return fmt.Errorf("failed to submit public key: [%v]", err)
 	}
 
+	go n.monitorKeepPublicKeySubmission(keepAddress, publicKey)
+
 	return nil
```

Comment:
There is very little happening in this function now - can we inline this code in `GenerateSignerForKeep` above? We can probably even eliminate the `Debugf` log as we have logs on Info level informing about the same.